Dockerfile

```
@@ -1,8 +1,3 @@
-FROM bitnami/kubectl:1.21.1 as kubectl
-FROM rancher/k3s:v1.21.1-k3s1 as k3s
-FROM mikefarah/yq:4.9.3 as yq2
-FROM jsonnetlibs/docsonnet:0.0.1 as docsonnet
-
 FROM golang:1.16.4 as base
```

You can bump that to 1.16.5
```suggestion
FROM golang:1.16.5 as base
```

---

This is impossible to keep track of 😌

---

I dropped the patch part.